File: demo/src/main/java/com/lcm/app/main/MainView.java
package com.lcm.app.main;



import android.content.Context;

import com.lcm.android.mvp.BaseView;

/**
 * ****************************************************************
 * Author: LCM
 * Date: 2017/5/23 下午12:59
 * Desc:
 * *****************************************************************
 */

public interface MainView extends BaseView {

    void onLoadSuccess(String str);

    Context getContext();

}

File: demo/src/main/java/com/lcm/app/main/MainView.kt
package com.lcm.app.main


import android.content.Context

import com.lcm.android.mvp.BaseView

/**
 * ****************************************************************
 * Author: LCM
 * Date: 2017/5/23 下午12:59
 * Desc:
 * *****************************************************************
 */

interface MainView : BaseView {

    fun getContext(): Context

    fun onLoadSuccess(str: String)

}
